kibbeh/public/locales/en/translation.json

```diff
@@ -294,4 +299,4 @@
       "spottedLife": "Planet with life spotted"
     }
   }
-}
+}
```

Comment:
No newline

Reply:
bero fuks u, atleast give proper reviews

Reply:
?

Reply:
do we need a newline here?